src/misc/KeyManager.js

```diff
@@ -168,6 +172,19 @@ class KeyManager {
 		return keycode + (ctrl ? "C" : "") + (alt ? "A" : "") + (shift ? "S" : "") + (meta ? "M" : "")
 	}
 
+	raiseEvent(keyCode: number, ctrl: ?boolean, alt: ?boolean, shift: ?boolean, meta: ?boolean): ?boolean {
```

Comment:
Not sure if it was better to expose a general `raiseEvent` as I did, or a more specific `openF1Help` mehtod.

Reply:
The more specific method would be preferred in this case, basically exposing the currently anonymous function that's defined under the F1 shortcut's exec key.